spec/features/users_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+
+feature "can sign up for an account" do
+  scenario "with valid attributes" do
+    visit new_user_path
+
+    fill_in "user_name", with: "Mary"
+    fill_in "user_email", with: "Mary@example.com"
+    fill_in "user_password", with: "secret"
+    click_on I18n.t("users.new.submit")
+
+    expect(page).to have_text(I18n.t("users.show.welcome", name: "Mary"))
+  end
+end
+
+feature "user can see their show page" do
+  scenario "signing up with valid attributes" do
```

Comment:
This is not really about signing up. I would remove line 16. Think about feature specs as a single `feature` block, with possible multiple `scenario` blocks.

So maybe change these two lines to be: `scenario "user can view their profile"`

This spec will need to change when you add authentication.

Reply:
Thanks Elle, you're right, not sure about my over arching feature statement though, see what you think.